### DISCUSSION TITLE
Reset to default settings.

### DISCUSSION THREAD

**Discussion by Monkeyman2111** [aaronwmorris/indi-allsky#1702] [General]
Hi all,      what is the easiest way to get all settings back to their default ?  I tried deleting everything in /etc/indi-allsky and re-run setup.sh but the previous setting survived.  

Thanks

**Comment by aaronwmorris** [comment]
The configs are stored in the database.  You can run the following...

```
source virtualenv/indi-allsky/bin/activate

./config.py flush

./config.py bootstrap
```

**Reply by Monkeyman2111** [reply]
That worked great, Thanks.
